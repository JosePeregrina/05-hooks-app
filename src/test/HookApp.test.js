import React from 'react'
import { shallow } from 'enzyme'
import '@testing-library/jest-dom'
import HookApp from '../HookApp'

describe('El componente <HookApp />', () => {
    test('debe de renderisarse correctamente', () => {
        const wrapper = shallow(<HookApp/>);

        expect(wrapper).toMatchSnapshot();
    })
})
