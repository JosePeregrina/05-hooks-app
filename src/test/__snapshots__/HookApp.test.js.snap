// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`El componente <HookApp /> debe de renderisarse correctamente 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
